tests/lambda_endpoints/test_batch_starter.py

```diff
@@ -154,7 +154,7 @@ def test_lambda_handler(session, s3_client):
             session,
             {"data_source": "swe", "data_type": "l1a", "descriptor": "sci"},
             dt.datetime(2024, 1, 1, 0, 0),
-            "v002",
+            "v001",
```

Comment:
Do we actually want these to be v001 now? Were we incorrectly setting these versions before, or what was going on that made this go down a version?